code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -109,7 +109,6 @@
 	amount_per_transfer_from_this = 12
 	volume = 12
 	ignore_flags = 1 //so you can medipen through hardsuits
-	reagent_flags = DRAWABLE
```

Comment:
actually, deleting this would cause even more issues as it'd make it default to using the same flags as regular autoinjectors, meaning people could refill them
instead make reagent_flags = NONE

Reply:
oh yeah i forgot

Reply:
Ammended!